### PR TITLE
Pinar Demirel Portfolio

### PR DESCRIPTION
**Pull request by pdetli**
https://pinardemirel.netlify.app/

### CODE REVIEW

File: code/TODO [hunk 1]
```diff
@@ -0,0 +1,4 @@
+highlighted text for skills and tech
+same margins for mobile?
+why the all icons in header are in same box? +
```
Comment (katiewu1) [inline code/TODO]: Maybe can delete the TODO document from GitHub? Don't think it's necessary to keep it here 😃

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./images/portfolio-image.png">
+    <link rel="icon" href="./images/diamond.png"> 
+    <title>Pinar Demirel's Portfolio </title>
+    <meta name="description" content="Frontend developer + PhD architect">
+
+    <!-- Facebook & Linkedin Meta Tags -->
```
Comment (katiewu1) [inline code/index.html:12]: cool that you got og tags 👍

File: code/index.html [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+        <div class="header-picture">
+            <img class="pinar-image" src="./images/pino.png" alt="image of Pinar">
+        </div>
+    </header>
+
+    <main>
+
+        <section class="summary-section">
+            <div class="container">
+                <p class="summary-text" tabindex="0">
```
Comment (katiewu1) [inline code/index.html]: do we need tabindex for summary-text? We can now focus on it but it's not interactive 🦊

File: code/index.html [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+                    <svg height="10" width="65">
+                     <line x1="0" y1="5" x2="50" y2="5" style="stroke:#FF4C29; stroke-width:3" />
+                     <circle cx="50" cy="5" r="5" fill= "#FF4C29"/>
+                    </svg>
+                     Hello! I'm Pinar – an ambitious frontend developer, currently on the way to be a junior frontend developer
```
Comment (katiewu1) [inline code/index.html]: Hello Pinar! 👋 😊

File: code/index.html [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+                <p class="tech"> HTML, CSS, Flexbox,
+                    <span class="highlighted-text">JavaScript ES6, JSX, React, Node.js,</span>
+                    React Hooks, Redux, Mongo DB, Web Accessibly, API:s, mob-programming, pair-programming, Github.
+                </p>
+
+            </div>
+        </section>
+
+        <section class="projects-section">
+            <h2 class="headline" tabindex="0"> FEATURED PROJECTS </h2>
```
Comment (katiewu1) [inline code/index.html]: same here, do we need the tabindex?

File: code/index.html [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+        <div class="projects-wrapper"> 
+            <div class="container project-container ">
+                <a href="https://github.com/pdetli/project-business-site" class="project-link" tabindex="0" target="_blank">
+                    <div class="project-card">
```
Comment (katiewu1) [inline code/index.html]: For even more semantic, maybe put an article tag instead of a div? 😺

File: code/index.html [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+                    
+                </div>
+            </div>
+            <div class="container project-container ">
+                <a href="https://github.com/pdetli/project-chatbot" class="project-link" tabindex="0" target="_blank">
+                    <div class="project-card">
+                        <div class="overlay"> </div>
+                        <img class="project-img" src="./images/chat-bot.png"
+                            alt="chat bot project preview">
+                        <p class="overlay-title">DJ BOT</p>
```
Comment (katiewu1) [inline code/index.html]: I really like this project 😉

File: code/index.html [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+
+        </div>
+
+        </section>
+
+
+        <section class="other-projects-section">
+            <h2 class="headline-other" tabindex="0"> OTHER PROJECTS </h2>
+            <div class=" container">
+            <div class="other-projects">
```
Comment (katiewu1) [inline code/index.html]: Maybe an article tag here as well? 💁

File: code/index.html [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+                <a aria-label="Link to project" class="projects-list" href="https://github.com/pdetli" target="_blank">
```
Comment (katiewu1) [inline code/index.html]: Nice that you use aria-label! 💯

File: code/index.html [hunk 10]
```diff
@@ -0,0 +1,352 @@
...
+                    <h2 class="project-title">Project Name  </h2>
+                        <p class="project-text">  Project information will be available soon..
+                        <span class="arrow"> >> </span>
+                    </p>
+                    <div class="tag-wrapper">
+                        <p aria-hidden="true">JavaScript ES6</p>
+                        <p aria-hidden="true">HTML5</p>
```
Comment (katiewu1) [inline code/index.html]: You use aria-hidden here for the tags but not for the project-card?

File: code/index.html [hunk 11]
```diff
@@ -0,0 +1,352 @@
...
+
+
+        <section class="skills-section">
+            <h2 class="headline" tabindex="0"> SKILLS </h2>
+            <div class="container"> 
+           
+                <div class="skills-display"> 
+                    <div class="skills-text">
+                        <h3 class="project-title" tabindex="0">CODE</h3>
+                        <ul tabindex="0">
```
Comment (katiewu1) [inline code/index.html]: Nice with the list! I used p-tags, so maybe I should change it to list instead 👍 😊

File: code/index.html [hunk 12]
```diff
@@ -0,0 +1,352 @@
...
+                    <img class="linkedin-icon-fm" src="./icons/linkedin.svg" alt="linkedin-icon">
+                    <p class="icon-text">LINKEDIN</p>
+                </a>
+                <a href="https://github.com/pdetli" target="blank">
+                    <img class="github-icon-fm" src="./icons/github.svg" alt="github-icon">
+                    <p class="icon-text">GITHUB</p>
+                </a>
+                <a href="https://stackoverflow.com/users/16699755/pinar-d" target="blank">
+                    <img class="sof-icon-fm" src="./icons/sof.svg" alt="sof-icon">
+                    <p class="icon-text">STACK OVERFLOW</p>
```
Comment (katiewu1) [inline code/index.html]: The text "stack overflow" is a bit too long, so the icons are not really so centered. Maybe try to break it in to two lines, or put the stack overflow-icon in the middle of those three icons (for example linkedin, stack overflow, github)? :octocat:

File: code/index.html [hunk 13]
```diff
@@ -0,0 +1,352 @@
...
+            </div>
+        <!-- </div> -->
+        </section>
+    </main>
+
+    <footer class="footer">
+        <h3 class="contact">CONTACT</h3>
+        <div class="contact-text">
+            <p>Pinar Demirel</p>
+            <p>+46 76 326 61 31 </p>
```
Comment (katiewu1) [inline code/index.html]: Maybe can put some ARIA here so the screen reader/voiceOver can read up your contact info 🎤

File: code/style.css [hunk 14]
```diff
@@ -0,0 +1,491 @@
+*{
+box-sizing: border-box;
+}
+
+
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    /* font-family: 'Roboto', sans-serif; */
```
Comment (katiewu1) [inline code/style.css]: forgot to remove line 9? 🐱

File: code/style.css [hunk 15]
```diff
@@ -0,0 +1,491 @@
...
+}
+
+
+section:nth-child(2n-1) {
```
Comment (katiewu1) [inline code/style.css:10]: Nice that you use nth-child! 💯 👍

File: code/style.css [hunk 16]
```diff
@@ -0,0 +1,491 @@
...
+}
+
+
+ .project-img {
+    display: block;
+    height: 280px;
+    transition: 0.5s ease; 
+    object-fit: cover;
+ 
+    /* to prevent scale issues for ie browser 280px setted for height  */
```
Comment (katiewu1) [inline code/style.css]: 👍 👍 👍

File: code/style.css [hunk 17]
```diff
@@ -0,0 +1,491 @@
...
+
+.my-thoughts-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+/* -------- SKILLS SECTION --------*/
+.skills-section {
```
Comment (katiewu1) [inline code/style.css]: the my-thoughts-section, skills-section and for-more-section have the same code. So maybe can put them together? Like 
```
.my-thoughts-section, .skill-section, .for-more-section { 
display: flex;
flex-direction: column;
align-items: center;
}
```

File: code/style.css [hunk 18]
```diff
@@ -0,0 +1,491 @@
...
+    padding-bottom: 30px;
+   
+}
+.linkedin-icon-fm:hover, .github-icon-fm:hover, .sof-icon-fm:hover {
+    filter: invert(13%) sepia(46%) saturate(1912%) hue-rotate(336deg) brightness(103%) contrast(102%);
+    /* to change the color when hover over */
+}
+
+.linkedin-icon-fm, .github-icon-fm, .sof-icon-fm {
+    filter: invert(85%) sepia(0%) saturate(7500%) hue-rotate(327deg) brightness(96%) contrast(104%);
```
Comment (katiewu1) [inline code/style.css]: 👍

**Review by katiewu1 — APPROVED** [review]
Hi Pinar! Really like your design of your portfolio (I went for yellow color too 😊 ).
It's easy to follow your code! There's some commented out code lines that maybe should be removed? And check for the indention/spaces too especially in css. 
In general well done! 💪  It was not easy to keep track of all the html and css code 💦  but so happy that we have our portfolios now 👏 😄

File: code/style.css (hunk 14 above)
Comment (pdetli) [reply code/style.css]: I was using that line to copy from there but seems that I forgot to remove it. 👍

**Comment by pdetli** [comment]
Thank you very much for this constructive and helpful code review! I did the changes as you pointed out!

**Pull request closed without merging**